### ISSUE TITLE
Stop condition for training

### ISSUE THREAD

**Issue by mbhall88** [nanoporetech/taiyaki#75]
Sorry if this is written somewhere in the docs and I have missed it. But what are the conditions for training finishing? 

In case it is useful, I have included plots of the loss and learning rate so far.

### Plot of my loss

![image](https://user-images.githubusercontent.com/20403931/79854566-8dd9a800-83c1-11ea-9d78-a7c56c444177.png)

### Plot of the learning rate

![image](https://user-images.githubusercontent.com/20403931/79854623-a649c280-83c1-11ea-950f-297cf4aff78c.png)

**Comment by marcus1487** [comment]
Training continues for `--niteration` steps (default value of `50000`). While it may appear at this scale that training loss has converged, you might set a lower maximum y-axis value to the training loss plot and see that the loss is continuing to drop.

**Comment by tmassingham-ont** [comment]
Hello.  The general strategy used by Taiyaki is to run for a fixed amount of time, performing validation analyses on the checkpoint models to choose the best.  Validating in this way is more flexible and can involve steps that would be difficult or inconvenient to incorporate into the to the training loss function (e.g. basecalling long reads, consensus accuracy).

**Comment by mbhall88** [comment]
Thanks!
